### PR TITLE
ui: add istio network filter

### PR DESCRIPTION
**Pull request by tomerlange**
This PR adds a new istio filter that shows k8s and istio objects which are related to network.

### CODE REVIEW

**Comment by tomerlange** [comment]
run skydive-cdd-overview-tests

**Comment by tomerlange** [comment]
run skydive-cdd-overview-tests

File: statics/js/components/topology.js [hunk 1]
```diff
@@ -613,6 +625,11 @@ var TopologyComponent = {
         self.addFilterK8sTypes(highlight, "storage", ["persistentvolume", "persistentvolumeclaim", "storageclass"]);
       }
 
+      if (self.isIstioEnabled()) {
+	self.addFilterIstioTypes(filter, "network", ["cluster", "container", "namespace", "pod", "virtualservice", "gateway"]);
```
Comment (safchain) [inline statics/js/components/topology.js]: these two lines need to be indented properly, a space is missing

Comment (tomerlange) [reply statics/js/components/topology.js]: DONE

**Pull request merged**